release-team/role-handbooks/docs/Release-Timeline.md

```diff
@@ -434,26 +435,36 @@ The Docs Lead is responsible for updating the Kubernetes API reference, kubectl,
 If you need help in building the reference documentation, reach out on Slack #sig-docs.
 
 ### Generate the reference documentation
+Update the generated documentation using a Python script ([Generating Reference Pages for Kubernetes Components and Tools](https://kubernetes.io/docs/contribute/generate-ref-docs/kubernetes-components/)).
 
-Update the generated documentation using a Python script ([Generating Reference Pages for Kubernetes Components and Tools](https://kubernetes.io/docs/contribute/generate-ref-docs/kubernetes-components/)). Before running the script, modify `reference.yml` to checkout the Kubernetes future release branch.
-
-![Update kubernetes release branch](pics/reference-yaml.png)
-
-The first step is to famialiarize yourself with the `website/updated-imported-docs/update-imported-docs.py` script
+The first step is to familiarize yourself with the `website/updated-imported-docs/update-imported-docs.py` script
 and the instructions for ([Generating Reference Pages for Kubernetes Components and Tools](https://kubernetes.io/docs/contribute/generate-ref-docs/quickstart/)).
 
-The reference documentation build depends upon a valid release tag.
+The reference documentation build depends upon a valid [release tag](https://github.com/kubernetes/kubernetes/tags).
 When a Kubernetes release candidate version tag such as `v1.19-rc.2` is created, you can consider building
 the reference documentation. Next:
 
 - Create a branch for this work based off of the `dev-[future release]` branch.
+```
+git checkout dev-[future-release]
```

Comment:
Does this code block need to be indented under the list item?

Reply:
it looks like this https://github.com/annajung/sig-release/blob/update-docs-role-handbook/release-team/role-handbooks/docs/Release-Timeline.md#generate-the-reference-documentation

I think it looks okay without being indented. wdyt?

Reply:
OK